spec/inputs/city_input_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+
+describe CityInput do
+  include RSpec::Rails::HelperExampleGroup
+
+  describe '#input' do
```

Comment:
The test setup looks intimidating! I will copy/paste that for my own custom input ☺️  Thanks!